core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2577,17 +2577,21 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
-  public static final PropertyKey WORKER_NETWORK_BLOCK_READER_THREADS_MAX =
-      new Builder(Name.WORKER_NETWORK_BLOCK_READER_THREADS_MAX)
-          .setDefaultValue(2048)
-          .setDescription("The maximum number of threads used to read blocks in the data server.")
+  public static final PropertyKey WORKER_NETWORK_BLOCK_IO_THREADS =
+      new Builder(Name.WORKER_NETWORK_BLOCK_IO_THREADS)
+          .setAlias(Name.WORKER_NETWORK_BLOCK_READER_THREADS_MAX,
+              Name.WORKER_NETWORK_BLOCK_WRITER_THREADS_MAX)
+          .setDefaultSupplier(() -> 2 * Runtime.getRuntime().availableProcessors(),
+              "The number of threads used to serve IO requests. Increasing this value to high can"
+                  + " cause the JVM to pause for extended periods of time.")
```

Comment:
(nit) it would be useful to mention how is the default value calculated in the description.